scripts/bootstrap-aio.yml

```diff
@@ -33,6 +33,20 @@
     rpco_deploy_ceph: "{{ lookup('env', 'DEPLOY_CEPH') }}"
     rpco_deploy_hardening: "{{ lookup('env', 'DEPLOY_HARDENING') }}"
 
+- name: Enable trusty-backports
+  hosts: all
```

Comment:
This should only target 'hosts'. If you target 'all' it will also execute against containers which is unnecessary.